### DISCUSSION TITLE
Search filter dropdown placeholder

### DISCUSSION THREAD

**Discussion by brambuz** [microsoft-search/pnp-modern-search#1590] [Q&A] [answered]
It's possible change the default placeholder in combo template?
![image](https://user-images.githubusercontent.com/25414801/144619967-4eb9246c-2b96-4704-8d92-1d787ffaa9a4.png)

I'm using version 4.4.1

**Comment by wobba** [comment, marked as answer]
Not today.

**Comment by brambuz** [comment]
Ok, thank you!

Do you have this feature in the roadmap?
